### DISCUSSION TITLE
Login information

### DISCUSSION THREAD

**Discussion by 24crot** [trickerer01/Ruxx#7] [General]
There are some content that requires logging into an account to view on RN site, is it possible to add login feature?

**Comment by trickerer01** [comment]
If some content only shown when logged in then I'm not aware of it. Need to know if it's worth implementing. Can you give an example (no links please)? Is it tags related? Meta tags maybe? I really have no idea.

Currently the only way to use login information is to log in via website and get `shm_user` / `shm_session` cookies to use within app.

**Reply by 24crot** [reply]
If you search for RastafarianSFM tags for example, the site will show additional content when you are logged in.

Is there any way to insert  `shm_user` / `shm_session` in the current app version?

**Reply by trickerer01** [reply]
I see.
This extra content seems to be hidden behind login wall in order to comply with regulations of the country where servers are located / website is registered, I assume US.
Thus, I won't be implementing a feature allowing to automate login process for RN. 

Currently the only way to use login information is to provide respective cookies using `Connection` -> `Headers / Cookies` window (F3 hotkey). Enter your `shm_user` and `shm_session` cookies in format `name:value`. If this doesn't work by itself you'll have to provide your `cf_clearance` cookie as well.
Cookies window supports `Enter` and `Del` hotkeys.

**Reply by 24crot** [reply]
I see, thanks for your reply.
